src/java.base/share/classes/java/security/Policy.java

```diff
@@ -27,72 +27,47 @@
- * in the Java runtime environment has permission to perform a
- * security-sensitive operation.
- *
- * <p> There is only one {@code Policy} object installed in the runtime at any
- * given time.  A {@code Policy} object can be installed by calling the
- * {@code setPolicy} method.  The installed {@code Policy} object can be
- * obtained by calling the {@code getPolicy} method.
- *
- * <p> If no {@code Policy} object has been installed in the runtime, a call to
- * {@code getPolicy} installs an instance of the default {@code Policy}
- * implementation (a default subclass implementation of this abstract class).
- * The default {@code Policy} implementation can be changed by setting the value
- * of the {@code policy.provider} security property to the fully qualified
- * name of the desired {@code Policy} subclass implementation. The system
- * class loader is used to load this class.
- *
- * <p> Application code can directly subclass {@code Policy} to provide a custom
- * implementation.  In addition, an instance of a {@code Policy} object can be
- * constructed by invoking one of the {@code getInstance} factory methods
- * with a standard type.  The default policy type is "JavaPolicy".
- *
- * <p> Once a {@code Policy} instance has been installed (either by default,
- * or by calling {@code setPolicy}), the Java runtime invokes its
- * {@code implies} method when it needs to
- * determine whether executing code (encapsulated in a ProtectionDomain)
- * can perform SecurityManager-protected operations.  How a {@code Policy}
- * object retrieves its policy data is up to the {@code Policy} implementation
- * itself. The policy data may be stored, for example, in a flat ASCII file,
- * in a serialized binary file of the {@code Policy} class, or in a database.
- *
- * <p> The {@code refresh} method causes the policy object to
- * refresh/reload its data.  This operation is implementation-dependent.
- * For example, if the policy object stores its data in configuration files,
- * calling {@code refresh} will cause it to re-read the configuration
- * policy files.  If a refresh operation is not supported, this method does
- * nothing.  Note that refreshed policy may not have an effect on classes
- * in a particular ProtectionDomain. This is dependent on the policy
- * provider's implementation of the {@code implies}
- * method and its PermissionCollection caching strategy.
  *
  * @author Roland Schemers
  * @author Gary Ellison
  * @since 1.2
- * @see java.security.Provider
- * @see java.security.ProtectionDomain
- * @see java.security.Permission
- * @see java.security.Security security properties
- * @deprecated This class is only useful in conjunction with
- *       {@linkplain SecurityManager the Security Manager}, which is deprecated
- *       and subject to removal in a future release. Consequently, this class
- *       is also deprecated and subject to removal. There is no replacement for
- *       the Security Manager or this class.
```

Comment:
Yes, we should be consistent. The deprecated text always comes first, and sometimes we want to say immediately why this class is not useful anymore in that text instead of later, further down. I don't know which is better. I guess I'll go with your suggestion just so it looks more like a normal class.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/09b6cd602c1f2c7998ed824878c5f5f43be69075